### ISSUE TITLE
[Feature Request] have a yes/no confirm prompt when clicking the "Done" button

### ISSUE THREAD

**Issue by jtroo** [tannerkrewson/drawphone#102]
I misclicked on "done" while going for a colour and couldn't go back to finish my drawing. This would be nice to help mitigate issues from misclicks.

**Comment by tannerkrewson** [comment]
good idea! i think this might be a little tedious. maybe a better way would be a button that lets you go back and finish your drawing if we're still waiting for other people to finish. not a perfect solution, but i think that would be a good compromise.
